geneticalgorythm/algorithm.py

```diff
@@ -41,8 +54,8 @@ class Algorithm:
     def __init__(self, classifier, *, population_size=10, min_attributes_count=10,
                  initial_attributes_standard_deviation=1, individuals_to_mutate_coefficient=0.015,
                  chromosomes_to_mutate_coefficient=0.005, cycles_count=2, loci_count=2,
-                 fitness_function=FitnessFunction.AveragePrecision, selection_algorithm=SelectionAlgorithm.foo):
-        self.selection_algorithm = selection_algorithm
```

Comment:
napiszę tu. Jest tu sobie taka funkcja 
```python
def _selection(self, population_with_fitness: PopulationWithFitness) -> List[Tuple[int, int]]:
        return [(0, 0) for _ in range(len(population_with_fitness))]
```
to możesz zmienić na 
```python
def _selection(self, population_with_fitness: PopulationWithFitness) -> List[Tuple[int, int]]:
        return [(0, 0) for _ in self.selection_algorithm(population_with_fitness, len(population_with_fitness)]
```
i zobaczyć czy jakoś działa to

Reply:
zrobione, działa program, nie wywala się